corehq/apps/userreports/sql/connection.py

```diff
@@ -1,3 +1,8 @@
+import random
+
+from corehq.sql_db.connections import connection_manager
+
+READ_REPLICA_ROLLOUT_FACTOR = 0.01  # 1%
```

Comment:
It would be nice to have this as a localsetting or setting in HQ UI. that way we wouldn't need a deploy to change this and we could have different values for other envs